### ISSUE TITLE
Improvement: Consider to include all polyfills inside model-viewer core.

### ISSUE THREAD

**Issue by Esteban26** [google/model-viewer#570]
we had a use case where I was trying to verify if model-viewer already exists in the DOM, otherwise i had to append the script, @cdata helped me with the validation, but even with that fix I don't know if all the polyfills related to model-viewer already exists in the current page, so would be better for us to have only one library, 

What I think is model-viewer should to decide if a polifyll is needed or not and call it.

**Comment by cdata** [comment]
If you load the Web Components polyfill (the one we show in our examples) synchronously, it will synchronously include the polyfill (_only_ if it is needed) using `document.write`. In this case, you will always be able to use the DOM APIs I described in https://github.com/GoogleWebComponents/model-viewer/issues/561#issuecomment-494233669 .

Otherwise, if you know you always want to include the polyfill, my recommendation is that you bundle the scripts together based on your own use case.

The reason we don't want to bundle the polyfills is because they are only needed for specific circumstances. Polyfills like the Web Components polyfill will detect if they should be loaded, and only fully load on certain browsers. If we bundled all of these things together, then every browser would pay the cost of loading the polyfills always.
